packages/manager/apps/dedicated/client/app/configuration/configuration.controller.js

```diff
@@ -67,18 +78,23 @@ angular.module('App').controller(
       return this.$q.when(sectionNames);
     }
 
+    getUser() {
+      return this.User.getUser().then((user) => {
```

Comment:
You should use `coreConfig.getUser()` instead,

Reply:
Done :)